src/PointwiseFunctions/Hydro/MassWeightedFluidItems.hpp

```diff
@@ -44,6 +46,20 @@ template <typename DataType>
 struct TildeDUnboundUtCriterion : db::SimpleTag {
   using type = Scalar<DataType>;
 };
+
+/// Contains TildeD * (coordinates in frame Fr).
+/// Label allows us to restrict the data to the x>0 (A) or x<0 (B)
+/// plane in grid coordinates (useful for NSNS). Use ObjectLabel::None
+/// to use the entire volume data.
+template <typename DataType, size_t Dim, ::domain::ObjectLabel Label,
+          typename Fr = Frame::Inertial>
+struct MassWeightedCoords : db::SimpleTag {
+  using type = tnsr::I<DataType, Dim, Fr>;
+  ::domain::ObjectLabel object_label = Label;
```

Comment:
Is this type alias needed?

Reply:
Probably not anymore; I put it at a time when I did not have a "name" function and thus Label was unused within the simple tag but necessary to recover the correct tag... I'll remove this.